### PR TITLE
Community Partner Logo - Technical Resources Community by Pooja Gera

### PR DESCRIPTION
**Pull request by pooja-gera**
- [x] Added the community partner logo to ``img/`` with the name ``technicalresourcescommunitybypoojagera.png``
- [x] Verified logo's correctness and looks for different screen sizes

### CODE REVIEW

File: .DS_Store
Comment (Animesh-Ghosh) [inline .DS_Store:1]: Please remove the `.DS_Store` files

**Review by akshayaurora — COMMENTED** [review]
Please remove .DS_Store

File: img/technicalresourcescommunitybypoojagera.png
Comment (Animesh-Ghosh) [inline img/technicalresourcescommunitybypoojagera.png:1]: Not sure if having a name of a person would be good.

**Review by Animesh-Ghosh — CHANGES_REQUESTED** [review]

**Review by akshayaurora — CHANGES_REQUESTED** [review]
Please remove your name from the community Logo.

**Comment by Animesh-Ghosh** [comment]
Closing this for now. You may re-open it after making the requested changes.

**Pull request closed without merging**